### ISSUE TITLE
Would you use a NEW non-profit, sanction-free platform?

### ISSUE THREAD

**Issue by sullyfischer** [1995parham/github-do-not-ban-us#1375]
Hi everyone. I am the creator of an Voten, an open-source project with more than 1k stars:
https://github.com/voten-co/voten

I'm thinking about forking https://github.com/gitlabhq/gitlabhq and starting an open-source AND sanction-free Git platform. My solution is just like Voten, I won't register it anywhere and won't host it on platforms such as Google Cloud which block IPs from countries such as Iran. 

What do you guys think? If I do so, will you use it?

**Comment by pistol88** [comment]
Governments are blocking resources also without any sanctions from other governments. For example, Linkedin has been blocked in Russia. Or non-profit Telegram, or just all websites about cars or JavaScript from Digitalocean network. No obviously sense, just blocked by stupid person with power.

We need "Dark Blockchain Github" without owners and user's accounts 🤔

**Comment by scvoet** [comment]
I don't think it is a good idea to open a new issue. It is totally unnecessary. It just distracts the attention of the topic. We should discuss in the original issue, what do you think?
------------
我并不认为新开一个issue是一个好主意，这是完全没有必要。它只是一味地分散了话题的关注度。我们应该在原issue讨论，你认为呢？

**Comment by sullyfischer** [comment]
@scvoet I did this because I'm not just suggesting an idea for others, I'm proposing to do it myself as I did with my previous open-source project.

**Comment by ovflowd** [comment]
Well, that would be interesting! I seen some people trying some open source versions. But why you just don't fork GitLab CE, I don't know?

Still, if the project scales up... I mean it depends if you want have a main running instance, or just code something and release it for free and open source 😅

**Comment by iAlex11** [comment]
That's a really good idea! There's a tool named **Salsa**, developed by Debian (https://wiki.debian.org/Salsa/), which did the same thing, and it works pretty well. If you decide to continue with this idea, I know Python and Swift, and a bit of JS, so if there's anything where I can help I'll happily do so. I'd like to contribute, as I'm tired of GitHub restricting people and the US sanctions, and it's time to fix this.

When Microsoft bought GitHub, everyone started moving to GitLab. Then, why won't anyone move to this platform as well?

This can be a really good opportunity to make a real platform run by the community and not by a single enterprise, so we can make a platform with real freedom of speech, free to use, for everyone.

**Comment by sullyfischer** [comment]
@ovflowd I will be having a version up. in fact, I'm not really thinking about having a version that people can install, although they can; because for that, they can already use Gitlab CE. 

I'm talking about the platform itself which people can use, without registering it anywhere, and features such as **end-to-end encryption for private repositories**. 

@iAlex11 thanks man. Let's keep in touch. We have to see how this topic goes and how much people are into it. One thing I learned from my previous project (Voten), is that no matter how much better your platform is, people are not going to migrate unless they're actually looking for another platform.

If all the stargazers of this repository are merely looking to push Github to restore everything, and not ready to leave it for another platform, then  starting a new one would be a waste.
